tasks/deployments/collect/deploy-aave-fee-collect-module.ts

```diff
@@ -0,0 +1,38 @@
+import '@nomiclabs/hardhat-ethers';
+import { task } from 'hardhat/config';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { deployWithVerify } from '../../helpers/utils';
+import { AaveFeeCollectModule__Factory, LensHub__factory } from '../../../typechain/factories';
+
+export let runtimeHRE: HardhatRuntimeEnvironment;
+
+const POOL_ADDRESSES_PROVIDER_ADDRESS_MUMBAI = '0x5343b5bA672Ae99d627A1C87866b8E53F47Db2E6';
+const POOL_ADDRESSES_PROVIDER_ADDRESS_POLYGON = '0xa97684ead0e402dC232d5A977953DF7ECBaB3CDb';
```

Comment:
I'd say let's pass them as a task param? otherwise we can do a small logic for choosing the constant depending on the network :thinking:

Reply:
Good call, I've added it as a param, and then have logic to set it to one of the defined addresses if nothing passed in, based on the network. Let me know what you think